src/Roslyn.Diagnostics.Analyzers/UnitTests/SymbolDeclaredEventMustBeGeneratedForSourceSymbolsTests.cs

```diff
@@ -1,16 +1,5 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using Microsoft.CodeAnalysis.Diagnostics;
-using Roslyn.Diagnostics.CSharp.Analyzers;
-using Roslyn.Diagnostics.VisualBasic.Analyzers;
-using Test.Utilities;
-using VerifyCS = Microsoft.CodeAnalysis.CSharp.Testing.XUnit.CodeFixVerifier<
-    Roslyn.Diagnostics.CSharp.Analyzers.CSharpSymbolDeclaredEventAnalyzer,
-    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
-using VerifyVB = Microsoft.CodeAnalysis.VisualBasic.Testing.XUnit.CodeFixVerifier<
-    Roslyn.Diagnostics.VisualBasic.Analyzers.BasicSymbolDeclaredEventAnalyzer,
-    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
```

Comment:
This is going to be annoying to add back if anyone ever adds tests here.

Reply:
I did not think we had many more legacy analyzers that we wanted to port, or at least they are not high enough priority, to warrant disabling this rule for test layer. If we end up feeling otherwise, it should be easy to revert this commit and execute other FixAll with this rule disabled for test layer.